appinventor/appengine/src/com/google/appinventor/client/widgets/properties/SubsetJSONPropertyEditor.java

```diff
@@ -53,14 +56,34 @@
 import com.google.gwt.user.client.ui.TreeItem;
 import com.google.gwt.user.client.ui.VerticalPanel;
 import com.google.gwt.user.client.ui.Widget;
+
 import java.util.HashMap;
 import java.util.List;
+import java.util.Objects;
 import java.util.Set;
 
+import java.util.logging.Logger;
+
+interface BeginnerToolkit extends ClientBundle {
+  BeginnerToolkit INSTANCE = GWT.create(BeginnerToolkit.class);
+
+  @Source("toolkit_beginner.json")
+  TextResource getToolkit();
+}
+
+interface IntermediateToolkit extends ClientBundle {
+  IntermediateToolkit INSTANCE = GWT.create(IntermediateToolkit.class);
+
+  @Source("toolkit_intermediate.json")
+  TextResource getToolkit();
+}
```

Comment:
Hmm. I am surprised this is allowed here but it would be better for these interfaces to be within the SubsetJSONParserEditor class.

Reply:
Sorry I meant SubsetJSONPropertyEditor (which is defined on the following line).